Modules/BenchPress.Azure/Public/Get-ResourceByType.ps1

```diff
@@ -127,6 +134,14 @@ function Get-ResourceByType {
       "AksCluster" {
         return Confirm-AksCluster -AKSName $ResourceName -ResourceGroupName $ResourceGroupName
       }
+      "AksNodePool" {
+        $params = @{
+          ResourceGroupName = $ResourceGroupName
+          ClusterName = $ClusterName
+          Name = $ResourceName
```

Comment:
Edit the block so that the `=` are aligned

Reply:
Done!